### PR TITLE
Binary setup

### PR DESCRIPTION
**Pull request by TinyPandas**
- Setup initial binaries for the project
  - Wally
  - Foreman
  - Selene
  - StyLua

### CODE REVIEW

**Comment by lolzmac** [comment]
LGTM!

**Comment by TinyPandas** [comment]
> LGTM!

You can leave a review. ;)

**Review by lolzmac — APPROVED** [review]

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-  pull_request:
   push:
-    branches: [ main ]
+  pull_request: [main, develop]
 
 jobs:
-  ci:
+  lint:
     runs-on: ubuntu-latest
     if: "!contains(github.event.head_commit.message, '[skip-ci]')"
```
Comment (TinyPandas) [inline .github/workflows/lint.yml:10]: To follow consistency should this be `'[skip-lint]'` instead of `'[skip-ci]'`?

Comment (cntkillme) [reply .github/workflows/lint.yml:10]: We could, but in the future if we add additional CI ideally this switch should also shut those down so this seems like the most generic and future-proof name.

Comment (TinyPandas) [reply .github/workflows/lint.yml:10]: Right, but this ensures format and such. Not anything relevant to CI.

Comment (cntkillme) [reply .github/workflows/lint.yml:10]: Sure we could split up linting from other forms of CI and have different badges, I'll change it

**Review by lolzmac — APPROVED** [review]

**Pull request merged**